pkgs/android-integration/termux-tools.nix

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) 2019-2024, see AUTHORS. Licensed under MIT License, see LICENSE.
+
+{ stdenvNoCC
+, fetchFromGitHub
+, autoreconfHook
+, makeWrapper
+, gnused
+, getopt
+, termux-am
+}:
+
+stdenvNoCC.mkDerivation rec {
+  name = "termux-tools";
+  version = "1.42.4";
+  src = fetchFromGitHub {
+    owner = "termux";
+    repo = "termux-tools";
+    rev = "v${version}";
+    sha256 = "sha256-LkkeaEQcY8HgunBYAg3Ymn5xYPvrGqGNCZTd/NyIOKY=";
+  };
+  nativeBuildInputs = [ autoreconfHook makeWrapper ];
+  propagatedInputs = [ termux-am ];
+
+  # https://github.com/termux/termux-tools/pull/95
+  patches = [ ./termux-tools.patch ];
+  postPatch = ''
+    substituteInPlace scripts/termux-setup-storage.in \
+      --replace @TERMUX_HOME@ /data/data/com.termux.nix/files/home/ \
+      --replace @TERMUX_APP_PACKAGE@ com.termux.nix
+    substituteInPlace scripts/termux-open.in \
+      --replace 'getopt ' '${getopt}/bin/getopt '
+    substituteInPlace \
+      scripts/termux-open.in \
+      scripts/termux-wake-lock.in \
+      scripts/termux-wake-unlock.in \
+      --replace @TERMUX_APP_PACKAGE@.app com.termux.app \
+      --replace @TERMUX_APP_PACKAGE@ com.termux.nix
+    substituteInPlace scripts/termux-reload-settings.in \
+      --replace @TERMUX_APP_PACKAGE@ com.termux.nix
+    ${gnused}/bin/sed -i 's|^am |${termux-am}/bin/am |' scripts/*
```

Comment:
Why do you use `substituteInPlace` above but gnused here?

Reply:
because substituteInPlace is the recommended Nix way, being more explicit and less surprising and everything...

... and then there are 6 calls to am that need to be patched to use the real path, but you can't just replace all `am`, it has to be at the beginning of the line, so I whip up a sed and use a regexp.